src/everwillow/statelib/meta.py

```
@@ -0,0 +1,98 @@
+
+from functools import partial
+import dataclasses
+import typing as tp
+import jax.tree_util as jtu
+from jaxtyping import PyTreeDef
+
+if tp.TYPE_CHECKING:
+    from everwillow.statelib.state import K, V, State, FrozenChainMap
+
+
+__all__ = ["MergeMetadata", "TreeDefMeta"]
```

Ruff (0.14.2)

Undefined name `MergeMetadata` in `__all__`

(F822)